src/main/resources/application-dev.yaml

```diff
@@ -7,7 +7,7 @@ spring:
   banner:
     location: classpath:banner.txt
   datasource:
-    url: jdbc:mysql://localhost:3306/trackexpensesapp?useUnicode=true&serverTimezone=UTC
+    url: jdbc:mysql://database:3306/trackexpensesapp?useUnicode=true&serverTimezone=UTC&useSSL=false
```

Comment:
Should not this be set via environment variable?

Reply:
Originally was in this way so I didn't change it. I also think that through environment variables would be better. I'll fix it